pages/icon/icons-list.tsx

```diff
@@ -9,15 +9,15 @@ import ScreenshotArea from '../utils/screenshot-area';
 
 import styles from './icons-list.scss';
 
-const sizes = ['small', 'normal', 'big', 'large'] as const;
+const sizes = ['small', 'normal', 'medium', 'big', 'large'] as const;
```

Comment:
This question is not relevant to this PR, but wondering if **big** is smaller than **large**

Reply:
`big` is smaller than `large`. The size ramp is following:
`small`: 16x16
`normal`: 16x16
`medium`: 20x20
`big`: 32x32
`large`: 48x48